reeds/function_libs/visualization/re_plots.py

```diff
@@ -102,7 +103,7 @@ def plot_replica_trace_maxContrib(trace_data, num_states, eoffs, svals, title, o
         plt.savefig(out_path)
         plt.close()  
 
-def plot_replica_trace_numSampled(trace_data, num_states, eoffs, svals, title, undersampling_thres, out_path=None):
```

Comment:
@eruijsena Great to remove this unnecessary variable. Could you please also remove it from the docstring. (comment just below function definition which tells you what the function takes as input)